packages/schematics/src/utils/name-utils.ts

```diff
@@ -40,3 +43,16 @@ function toCapitalCase(s: string): string {
 export function findModuleParent(modulePath) {
   return path.dirname(modulePath);
 }
+
+/**
+ * Simple logic to ensure the name is a 'plural'.
+ * @see https://github.com/blakeembrey/pluralize/blob/master/pluralize.js for complex pluralizations
+ */
+export function pluralize(name: string): string {
```

Comment:
isn't used. remove it.

in general, autopluralizers are problematic (as say Rails showed)